crates/configuration/src/parachain.rs

```diff
@@ -223,7 +229,7 @@ impl<A> ParachainConfigBuilder<A> {
             default_image: self.config.default_image.clone(),
             default_resources: self.config.default_resources.clone(),
             default_db_snapshot: self.config.default_db_snapshot.clone(),
-            default_args: self.config.default_args.clone(),
+            default_args: self.config.default_args().into_iter().cloned().collect(),
```

Comment:
Not needed

```suggestion
            default_args: self.config.clone(),
```

Reply:
That does not seem correct 👀 
Why clone the config in the default args (type mismatch)

Reply:
typo error ```self.config.default_args.clone()```